src/Host/Client/Impl/BrokerServices/RemoteUriResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.R.Host.Client.BrokerServices {
+    public class RemoteUriResponse {
```

Comment:
This looks like trimmed-down `HttpResponseMessage` - why not use that?

Also, this assembly is intended for DTOs that are shared between the broker and the client to use at the API boundary. Looks like the broker doesn't actually use this anywhere, so it shouldn't be here.